.cirrus.yml

```diff
@@ -0,0 +1,120 @@
+gcp_credentials: ENCRYPTED[!17c59813193e86cc337bce848b358412b90f50bc5fe1b8b39d363cdf14a41ebe76cfba0482e7f81d076994b9f6dbfb4c!]
+ 
+env:
+  ARTIFACTORY_URL: ENCRYPTED[!2f8fa307d3289faa0aa6791f18b961627ae44f1ef46b136e1a1e63b0b4c86454dbb25520d49b339e2d50a1e1e5f95c88!]
+  ARTIFACTORY_PRIVATE_USERNAME: private-reader
+  ARTIFACTORY_PRIVATE_PASSWORD: ENCRYPTED[!921e2792ce1fc164aaea1146ab2478e7aefd8aaa87022ca745adccee4deaa470bb883ad3066738fceb37622f239296a7!]
+  ARTIFACTORY_DEPLOY_USERNAME: public-qa-deployer
+  ARTIFACTORY_DEPLOY_PASSWORD: ENCRYPTED[!ee8f6410a42b81c6c91c7b760e7be7796a7774b6e6e6acf3ff2ecf8c7c04c732865a0300673ea41155c0d19a989c0a5a!]
+  #Possible values for ARTIFACTORY_DEPLOY_REPO: sonarsource-private-qa, sonarsource-public-qa
+  ARTIFACTORY_DEPLOY_REPO: sonarsource-public-qa
+  ARTIFACTORY_API_KEY: ENCRYPTED[!f9526e763214dcd6f0cac5fbf712664bcf05395cc5c8f9f122ccf3a4c6d42c2bc809bff8a6d9904c935a47a05676682c!]
+  # burgr notification
+  BURGR_URL: ENCRYPTED[!c7e294da94762d7bac144abef6310c5db300c95979daed4454ca977776bfd5edeb557e1237e3aa8ed722336243af2d78!]
+  BURGR_USERNAME: ENCRYPTED[!b29ddc7610116de511e74bec9a93ad9b8a20ac217a0852e94a96d0066e6e822b95e7bc1fe152afb707f16b70605fddd3!]
+  BURGR_PASSWORD: ENCRYPTED[!83e130718e92b8c9de7c5226355f730e55fb46e45869149a9223e724bb99656878ef9684c5f8cfef434aa716e87f4cf2!]
+container_definition: &CONTAINER_DEFINITION
+  cluster_name: cirrus-ci-lt-cluster
+  zone: us-central1-a
+  namespace: default
+
+build_task:
+  gke_container:
+    image: gcr.io/language-team/base:v1
+    <<: *CONTAINER_DEFINITION
+    cpu: 2
+    memory: 2G
+  env:
+    GITHUB_TOKEN: ENCRYPTED[!a4048528b1a03f194719e3295c10e46c1e197c706b3d7c71dfc93540535aa9012ae61c3b66d0d6c6901b6a5e0bd84c68!]
+    # analysis on next
+    SONAR_TOKEN: ENCRYPTED[!5ba7cbb5bf9d168de69bcd444d9e884c9cf664be1115640cc64e49df6d241c309a87fc527cab533c08f289b167187017!]
+    SONAR_HOST_URL: https://next.sonarqube.com/sonarqube
+    #allow deployment of pull request artifacts to repox
+    DEPLOY_PULL_REQUEST: true
+    CIRRUS_CLONE_DEPTH: 50
+  maven_cache:
+    folder: ~/.m2/repository 
+  build_script:
+    - source cirrus-env BUILD
+    - regular_mvn_build_deploy_analyse
+  cleanup_before_cache_script:
+    - rm -rf "$HOME/.m2/repository/org/sonarsource/"
```

Comment:
Why do we use `$HOME/.m2/repository` here and `~/.m2/repository` 5 line above?
Why do we use `$CIRRUS_WORKING_DIR/.m2/repository` in sonar-tsql?

Reply:
~ and $HOME are the same but indeed a little alignment here would be good

Reply:
second comment is the key, we should now use $CIRRUS_WORKING_DIR